ansible-tests/validations/check_openstack_endpoints.yaml

```diff
@@ -0,0 +1,35 @@
+---
+- hosts: undercloud
+  vars:
+    metadata:
+      name: Check access to various openstack services
+      description: >
+        This will attempt to connect to each service endpoint defined in
+        your hosts file's controller_vip variable.
+      groups:
+      - post-deployment
+  tasks:
+  - name: Load the overcloud credentials
+    overcloudrc: path=/home/stack/overcloudrc
+
+  # Check connectivity to horizon
+  - name: Check Horizon
+    # TODO: get Horizon endpoint from Keystone
+    uri: url=http://{{public_vip}}
+
+  # Check that we can obtain an auth token from horizon
+  - name: Check Keystone
```

Comment:
Huh. So, this is an Ansible 1 vs. 2 issue again. Works fine with 2 but I see the crash with 1.

It actually looks like a bug in the "uri" module. Even if I pass a string (not a dict) to "body", something will process it as a dictionary and cause the error.